### ISSUE TITLE
Unable to install h3 using pip

### ISSUE THREAD

**Issue by Tanay0510** [uber/h3-py#101]

**Comment by Tanay0510** [comment]
Collecting h3
  Using cached h3-3.4.3.tar.gz (10 kB)
Building wheels for collected packages: h3
  Building wheel for h3 (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/tanaybhardwaj/opt/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py'"'"'; __file__='"'"'/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-wheel-wo1wnwiw
       cwd: /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/
  Complete output (117 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.9-x86_64-3.7
  creating build/lib.macosx-10.9-x86_64-3.7/h3
  copying h3/h3.py -> build/lib.macosx-10.9-x86_64-3.7/h3
  copying h3/__init__.py -> build/lib.macosx-10.9-x86_64-3.7/h3
  running build_ext
  + VERSION=v3.4.2
  + IS_64BITS=True
  + '[' '' == v3.4.2 ']'
  + command -v cmake
  + mkdir -p h3/out
  + rm -rf h3c
  + git clone https://github.com/uber/h3.git h3c
  Cloning into 'h3c'...
  + pushd h3c
  /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3
  + git pull origin master --tags
  From https://github.com/uber/h3
   * branch            master     -> FETCH_HEAD
  Already up to date.
  + git checkout v3.4.2
  Note: switching to 'v3.4.2'.
  
  You are in 'detached HEAD' state. You can look around, make experimental
  changes and commit them, and you can discard any commits you make in this
  state without impacting any branches by switching back to a branch.
  
  If you want to create a new branch to retain commits you create, you may
  do so (now or later) by using -c with the switch command. Example:
  
    git switch -c <new-branch-name>
  
  Or undo this operation with:
  
    git switch -
  
  Turn off this advice by setting config variable advice.detachedHead to false
  
  HEAD is now at d610a22 Merge pull request #194 from isaacbrodsky/version-3.4.2
  + '[' Windows_NT '!=' '' ']'
  + command -v make
  + command -v cc
  + cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
  -- The C compiler identification is Clang 4.0.1
  -- Check for working C compiler: /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang
  -- Check for working C compiler: /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang - broken
  CMake Error at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/CMakeTestCCompiler.cmake:60 (message):
    The C compiler
  
      "/Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang"
  
    is not able to compile a simple test program.
  
    It fails with the following output:
  
      Change Dir: /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp
  
      Run Build Command(s):/usr/local/bin/gmake cmTC_1706a/fast && /usr/local/bin/gmake  -f CMakeFiles/cmTC_1706a.dir/build.make CMakeFiles/cmTC_1706a.dir/build
      gmake[1]: Entering directory '/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp'
      Building C object CMakeFiles/cmTC_1706a.dir/testCCompiler.c.o
      /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang   -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk   -std=gnu99 -o CMakeFiles/cmTC_1706a.dir/testCCompiler.c.o   -c /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp/testCCompiler.c
      Linking C executable cmTC_1706a
      /usr/local/Cellar/cmake/3.17.2/bin/cmake -E cmake_link_script CMakeFiles/cmTC_1706a.dir/link.txt --verbose=1
      /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs  CMakeFiles/cmTC_1706a.dir/testCCompiler.c.o  -o cmTC_1706a
      ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd, file was built for unsupported file format ( 0x2D 0x2D 0x2D 0x20 0x21 0x74 0x61 0x70 0x69 0x2D 0x74 0x62 0x64 0x2D 0x76 0x33 ) which is not the architecture being linked (x86_64): /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd
      ld: dynamic main executables must link with libSystem.dylib for architecture x86_64
      clang-4.0: error: linker command failed with exit code 1 (use -v to see invocation)
      gmake[1]: *** [CMakeFiles/cmTC_1706a.dir/build.make:107: cmTC_1706a] Error 1
      gmake[1]: Leaving directory '/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp'
      gmake: *** [Makefile:141: cmTC_1706a/fast] Error 2
  
  
  
  
  
    CMake will not be able to correctly generate this project.
  Call Stack (most recent call first):
    CMakeLists.txt:54 (project)
  
  
  -- Configuring incomplete, errors occurred!
  See also "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeOutput.log".
  See also "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeError.log".
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py", line 65, in <module>
      distclass=BinaryDistribution)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/site-packages/setuptools/__init__.py", line 145, in setup
      return distutils.core.setup(**attrs)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 966, in run_commands
      self.run_command(cmd)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/site-packages/wheel/bdist_wheel.py", line 192, in run
      self.run_command('build')
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/command/build.py", line 135, in run
      self.run_command(cmd_name)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py", line 25, in run
      install_h3(h3_version)
    File "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py", line 18, in install_h3
      subprocess.check_call('bash ./.install.sh {} {}'.format(h3_version, is_64bits), shell=True)
    File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/subprocess.py", line 347, in check_call
      raise CalledProcessError(retcode, cmd)
  subprocess.CalledProcessError: Command 'bash ./.install.sh v3.4.2 True' returned non-zero exit status 1.
  ----------------------------------------
  ERROR: Failed building wheel for h3
  Running setup.py clean for h3
Failed to build h3
Installing collected packages: h3
    Running setup.py install for h3 ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/tanaybhardwaj/opt/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py'"'"'; __file__='"'"'/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-record-w0h6reab/install-record.txt --single-version-externally-managed --compile --install-headers /Users/tanaybhardwaj/opt/anaconda3/include/python3.7m/h3
         cwd: /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/
    Complete output (119 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.9-x86_64-3.7
    creating build/lib.macosx-10.9-x86_64-3.7/h3
    copying h3/h3.py -> build/lib.macosx-10.9-x86_64-3.7/h3
    copying h3/__init__.py -> build/lib.macosx-10.9-x86_64-3.7/h3
    running build_ext
    + VERSION=v3.4.2
    + IS_64BITS=True
    + '[' '' == v3.4.2 ']'
    + command -v cmake
    + mkdir -p h3/out
    + rm -rf h3c
    + git clone https://github.com/uber/h3.git h3c
    Cloning into 'h3c'...
    + pushd h3c
    /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3
    + git pull origin master --tags
    From https://github.com/uber/h3
     * branch            master     -> FETCH_HEAD
    Already up to date.
    + git checkout v3.4.2
    Note: switching to 'v3.4.2'.
    
    You are in 'detached HEAD' state. You can look around, make experimental
    changes and commit them, and you can discard any commits you make in this
    state without impacting any branches by switching back to a branch.
    
    If you want to create a new branch to retain commits you create, you may
    do so (now or later) by using -c with the switch command. Example:
    
      git switch -c <new-branch-name>
    
    Or undo this operation with:
    
      git switch -
    
    Turn off this advice by setting config variable advice.detachedHead to false
    
    HEAD is now at d610a22 Merge pull request #194 from isaacbrodsky/version-3.4.2
    + '[' Windows_NT '!=' '' ']'
    + command -v make
    + command -v cc
    + cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
    -- The C compiler identification is Clang 4.0.1
    -- Check for working C compiler: /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang
    -- Check for working C compiler: /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang - broken
    CMake Error at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/CMakeTestCCompiler.cmake:60 (message):
      The C compiler
    
        "/Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang"
    
      is not able to compile a simple test program.
    
      It fails with the following output:
    
        Change Dir: /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp
    
        Run Build Command(s):/usr/local/bin/gmake cmTC_3acbe/fast && /usr/local/bin/gmake  -f CMakeFiles/cmTC_3acbe.dir/build.make CMakeFiles/cmTC_3acbe.dir/build
        gmake[1]: Entering directory '/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp'
        Building C object CMakeFiles/cmTC_3acbe.dir/testCCompiler.c.o
        /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang   -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk   -std=gnu99 -o CMakeFiles/cmTC_3acbe.dir/testCCompiler.c.o   -c /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp/testCCompiler.c
        Linking C executable cmTC_3acbe
        /usr/local/Cellar/cmake/3.17.2/bin/cmake -E cmake_link_script CMakeFiles/cmTC_3acbe.dir/link.txt --verbose=1
        /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs  CMakeFiles/cmTC_3acbe.dir/testCCompiler.c.o  -o cmTC_3acbe
        ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd, file was built for unsupported file format ( 0x2D 0x2D 0x2D 0x20 0x21 0x74 0x61 0x70 0x69 0x2D 0x74 0x62 0x64 0x2D 0x76 0x33 ) which is not the architecture being linked (x86_64): /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd
        ld: dynamic main executables must link with libSystem.dylib for architecture x86_64
        clang-4.0: error: linker command failed with exit code 1 (use -v to see invocation)
        gmake[1]: *** [CMakeFiles/cmTC_3acbe.dir/build.make:107: cmTC_3acbe] Error 1
        gmake[1]: Leaving directory '/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeTmp'
        gmake: *** [Makefile:141: cmTC_3acbe/fast] Error 2
    
    
    
    
    
      CMake will not be able to correctly generate this project.
    Call Stack (most recent call first):
      CMakeLists.txt:54 (project)
    
    
    -- Configuring incomplete, errors occurred!
    See also "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeOutput.log".
    See also "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/h3c/CMakeFiles/CMakeError.log".
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py", line 65, in <module>
        distclass=BinaryDistribution)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/site-packages/setuptools/__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/command/install.py", line 545, in run
        self.run_command('build')
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py", line 25, in run
        install_h3(h3_version)
      File "/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py", line 18, in install_h3
        subprocess.check_call('bash ./.install.sh {} {}'.format(h3_version, is_64bits), shell=True)
      File "/Users/tanaybhardwaj/opt/anaconda3/lib/python3.7/subprocess.py", line 347, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command 'bash ./.install.sh v3.4.2 True' returned non-zero exit status 1.
    ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/tanaybhardwaj/opt/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py'"'"'; __file__='"'"'/private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-install-y7r7990f/h3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/24/tcjk47xx2h351wxs2mbybm640000gn/T/pip-record-w0h6reab/install-record.txt --single-version-externally-managed --compile --install-headers /Users/tanaybhardwaj/opt/anaconda3/include/python3.7m/h3 Check the logs for full command output.

**Comment by dfellis** [comment]
```
-- Check for working C compiler: /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang
-- Check for working C compiler: /Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang - broken
CMake Error at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/CMakeTestCCompiler.cmake:60 (message):
  The C compiler

    "/Users/tanaybhardwaj/opt/anaconda3/bin/x86_64-apple-darwin13.4.0-clang"

  is not able to compile a simple test program.
```

This is the key. Your laptop's C compiler doesn't work, so it cannot build the C component of h3-py.

You need to get a working C compiler before you can install.

**Comment by Tanay0510** [comment]
But I did which cc, which cmake and which git and they are already there. 

which cc
/usr/bin/cc
(base) Tanays-MacBook-Pro:~ tanaybhardwaj$ which cmake
/Users/tanaybhardwaj/opt/anaconda3/bin/cmake
(base) Tanays-MacBook-Pro:~ tanaybhardwaj$ which git
/usr/local/bin/git

**Comment by dfellis** [comment]
You have a compiler, but it does not function. That's what the error message is saying. Something is significantly broken with your compiler on your laptop.

**Comment by Tanay0510** [comment]
How can I reinstall it and make it work? I don't know where to look. :(

**Comment by Tanay0510** [comment]
I am able to load it using brew install but does not install using pip install. I need to use it for a program. But whenever I run it after importing he, it give me attribute error, geo_to_h3 not found

**Comment by dfellis** [comment]
That error is happening because the C component is not being built here.

I don't use a Mac anymore, so I can't help you much there, but I would look into whatever this `anaconda3` package manager is that you're using for clang and see about reinstalling from there, or switching to [Apple's official XCode CLI tools](https://stackoverflow.com/questions/9329243/how-to-install-xcode-command-line-tools) that includes clang (a C compiler) and not using this "anaconda3" thing for that.

**Comment by Tanay0510** [comment]
It worked. I don't know what it did. But it worked. 

export CC=/usr/bin/cc
export CXX=/usr/bin/c++

**Comment by Tanay0510** [comment]
BUT now it is giving this error. 

module 'h3' has no attribute 'geo_to_h3'", 'occurred at index 313626

**Comment by dfellis** [comment]
You likely still have a cached version of the broken H3 in your virtualenv. I would recommend blowing it away and rerunning `pip install` from scratch.

**Comment by dfellis** [comment]
> It worked. I don't know what it did. But it worked.
> 
> export CC=/usr/bin/cc
> export CXX=/usr/bin/c++

This forced it to use the standard C compiler for your laptop instead of whatever that `anaconda3` version is.

**Comment by Tanay0510** [comment]
I did remove the cached but I am still getting that error. Attribute error

**Comment by dfellis** [comment]
Are you importing it as `from h3 import h3`?

**Comment by ajfriend** [comment]
We just released v3.6.1, which should hopefully fix these install issues. Could you please try installing with `pip install h3==3.6.1` and let us know how it works?

I'll close this for now. Please re-open if the issue persists!
